core-web/libs/template-builder/src/lib/components/template-builder/utils/gridstack-options.ts

```diff
@@ -36,21 +51,22 @@ function isARowWidget(el: Element): boolean {
 }
 
 export const subGridOptions: GridStackOptions = {
-    cellHeight: 224,
+    cellHeight: GRID_STACK_BOX_HEIGHT,
     column: 'auto',
-    margin: `${GRID_STACK_MARGIN}${GRID_STACK_UNIT}`,
+    margin: `${GRID_STACK_MARGIN_VERTICAL}${GRID_STACK_UNIT} ${GRID_STACK_MARGIN_HORIZONTAL}${GRID_STACK_UNIT}`,
```

Comment:
This takes a `marginUnit` I think we should use it.

Reply:
That's right, but it says that `marginUnit` property is an internal, and it setted by GridStack when the margin is a `string`